docs/api.md

```diff
@@ -456,6 +456,17 @@ Each *Object* is required to have the following properties:
   example, specifying `myapp` would cause URLs such as `myapp://path` to be opened with the app.
   Maps to the `CFBundleURLSchemes` metadata property.
 
+##### `usageDescription`
```

Comment:
It's not clear reading these docs that this should be an object

Reply:
Yep, I forgot the type. I added a type and an example in https://github.com/electron/electron-packager/pull/1049/commits/525c04d629e630b6f449e842b1569b18dd205268.

I really need to figure out a better way to deal with the API docs. Perhaps TypeScript definitions + typedoc.